webui/react/src/components/TaskTable.tsx

```diff
@@ -54,15 +57,68 @@ export const tableRowClickHandler = (record: AnyTask): {onClick?: MouseEventHand
 });
 
 const TaskTable: React.FC<Props> = ({ tasks }: Props) => {
+  const [ selectedRowKeys, setSelectedRowKeys ] = useState([]);
+
+  const showBatch = selectedRowKeys.length !== 0;
```

Comment:
thought: can batch actions, filters/ etc easily sit outside the table component may be on the page? 
should they?

Reply:
great questions, let's discuss this in our webui arch sync